app/src/main/res/layout/drink_card.xml

```diff
@@ -95,17 +95,17 @@
             android:layout_marginStart="@dimen/shortdescription_marginLeft"
             android:layout_marginTop="@dimen/shortdescription_marginTop"
             android:textAppearance="@style/drink_card_beerDescription"
-            android:maxLines="4"
+            android:maxLines="@string/shortdescription_maxLines"
             android:ellipsize="end"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintLeft_toRightOf="@id/drink_image"
             app:layout_constraintRight_toRightOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintVertical_bias="0.0"
-            tools:text="Descrizione" />
+            app:layout_constraintVertical_bias="@dimen/shortdescription_verticalBias"
+            tools:text="@string/drinkshortdescription_text"  />
```

Comment:
if we want to use the underscore, then we need to use it everywhere not only into a part of the string name